internal/dcache/rpc/server/handler.go

```diff
@@ -303,11 +303,8 @@ func (rv *rvInfo) incReservedSpace(bytes int64) {
 // Decrement the reserved space for this RV.
 func (rv *rvInfo) decReservedSpace(bytes int64) {
 	common.Assert(bytes > 0)
-	//
-	// TODO: Uncomment this only after clustermanager joinMV() correctly reserves space for MV replica.
-	//
-	// rv.reservedSpace.Add(-bytes)
-	// common.Assert(rv.reservedSpace.Load() >= 0, rv.rvName, rv.reservedSpace.Load())
+	rv.reservedSpace.Add(-bytes)
+	common.Assert(rv.reservedSpace.Load() >= 0, rv.rvName, rv.reservedSpace.Load())
```

Comment:
what kind of validation have you done?
this needs to be thoroughly tested

Reply:
Verified through JoinMV workflow for degraded MV.